docs/getting-started/installation.md

```diff
@@ -25,6 +25,25 @@ docker run -d \
 
 {% endtab %}
 
+
+{% tab title="Compose" %}
```

Comment:
Add an additional line break here to match the formatting of the rest of the document.